tests/travis/test.yaml

```diff
@@ -9,7 +9,7 @@
   skip: ["image.py", "image_rgba.py", "ajax_source.py"]
 - path: "plotting/server"
   type: server
-  skip: ["image.py", "image_rgba.py", "remote_image.py", "serversource.py", "server_source_upload.py", "embedding_example.py", "abstractrender.py", "census.py"]
```

Comment:
IIRC, all examples using ServerDataSource and BlazeDataSource should be broken. Wouldn't be the case to either use a blaze.server instance to make them work or remove them for now?